docs/guide/quickstart.md

```
@@ -50,145 +49,300 @@ async function main() {
+for await (const event of client.parseParquetStream(file)) {
+  switch (event.type) {
+    case 'start':
+      console.log(`Processing ~${event.total_estimate} entities`);
+      break;
+
+    case 'batch':
+      // Add meshes to renderer as they arrive
+      await renderer.addMeshes(event.meshes);
+      console.log(`Batch ${event.batch_number}: ${event.mesh_count} meshes`);
+      break;
+
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)